### ISSUE TITLE
Older Worlds Crash on chunk generation when using the new version of the mod

### ISSUE THREAD

**Issue by SirAron111** [McJty/LostCities#182]
Here is the crash https://pastebin.com/nFTPgpQC
I can load the old world up i made sure to turn the config optimizedChunkgen setting in the general config to false as I do have NEID installed. (Tested it with this setting on true still didn’t work)
So like I say I can load the world but the moment i try to generate new chunks this crash happens
PS: also tested it with generating completely new world this also doesnt work her is the crash right at the start here is the crash for that https://pastebin.com/FA99Q9Mm

**Comment by SirAron111** [comment]
Additional Information
1.	Forge version used: 14.23.5.2811
2.	Modpack I updated (My Modpack) Dimension Zero
3.	Previous version of the lost cities installed: LostCities - 1.12-2.0.13
4.	Version I tried to update to that made this crash on chunk gen or new world gen happen: LostCities - 1.12-2.0.14
5.	Tested with the config setting that was added turned to false and turned to true
6.	Default profile set to be used in the config (worked perfectly before) “chisel”
